step-templates/liquibase-apply-changeset.json

```diff
@@ -22,7 +22,7 @@
ion -DownloadFolder $workingFolder\n\n    # Download and extract java and add it to PATH environment variable\n    Get-Java\n\n    # Get the driver\n    $driverPath = Get-DatabaseJar -DatabaseType $liquibaseDatabaseType\n\n\tif (![string]::IsNullOrEmpty($driverPath))\n    {\n    \t# Add to arguments\n    \t$liquibaseArguments += \"--classpath=$driverPath\"\n    }\n}\nelse\n{\n    if (![string]::IsNullOrEmpty($liquibaseClassPath))\n    {\n    \t$liquibaseArguments += \"--classpath=$liquibaseClassPath\"\n    }\n}\n\n# Check to see if liquibase path has been defined\nif ([string]::IsNullOrEmpty($liquibaseExecutablePath))\n{\n    # Assign root\n    $liquibaseExecutablePath = $PSSCriptRoot\n}\n\n# Get the executable location\n$liquibaseExecutable = Get-ChildItem -Path $liquibaseExecutablePath -Recurse | Where-Object {$_.Name -eq \"liquibase.bat\"}\n\n# Add path to current session\n$env:PATH += \";$($liquibaseExecutable.Directory)\"\n\n# Check to make sure it was found\nif ([string]::IsNullOrEmpty($liquibaseExecutable))\n{\n    # Could not find the executable\n    Write-Error \"Unable to find liquibase.bat in $PSScriptRoot or subfolders.\"\n}\n\n# Add argument for driver\n#$databaseDriver = Get-DatabaseDriverName -DatabaseType $liquibaseDatabaseType\n#if (![string]::IsNullOrEmpty($databaseDriver))\n#{\n#\t$liquibaseArguments += \"--driver=$databaseDriver\"\n#}\n\n# Add connection Url\n$connectionUrl = Get-ConnectionUrl -DatabaseType $liquibaseDatabaseType -ServerPort $liquibaseServerPort -ServerName $liquibaseServerName -DatabaseName $liquibaseDatabaseName -QueryStringParameters $liquibaseQueryStringParameters\n$liquibaseArguments += \"--url=$connectionUrl\"\n\n# Add Username and password\n$liquibaseArguments += \"--username=$liquibaseUsername\"\n$liquibaseArguments += \"--password=`\"$liquibasePassword`\"\"\n\n# Set the location to where the file is\nSet-Location -Path $changeLogFile.Directory\n\n# Check to see if it should run or just report\nif ($liquibaseReport -eq $true)\n{\n    # Set error action preference - updateSQL writes to stderr so this is to prevent errors from showing up\n    $ErrorActionPreference = \"SilentlyContinue\"\n    \n    # Add just report\n    $liquibaseArguments += \"updateSQL\"\n    \n    # Execute liquibase\n    $liquibaseProcess = Start-Process -FilePath $liquibaseExecutable.FullName -ArgumentList $liquibaseArguments -RedirectStandardError \"$PSScriptRoot\\stderr.txt\" -RedirectStandardOutput \"$PSScriptRoot\\ChangeSet.sql\" -Passthru -Wait\n\t    \n    # Display standard error\n    foreach ($line in (Get-Content -Path \"$PSScriptRoot\\stderr.txt\"))\n    {\n    \t# Display\n        Write-Host \"$line\"\n    }\n    \n    # Check exit code\n    if ($liquibaseProcess.ExitCode -eq 0)\n    {\n    \t# Attach artifact\n        New-OctopusArtifact -Path \"$PSScriptRoot\\ChangeSet.sql\" -Name \"ChangeSet.sql\"\n    }\n}\nelse\n{\n    $liquibaseArguments += \"update\"\n    & $liquibaseExecutable.FullName $liquibaseArguments\n}\n\n"
```

Comment:
Couple of things to check

1. This line can be safely removed:
```ps
#Invoke-WebRequest -Uri "https://repo1.maven.org/maven2/org/liquibase/ext/liquibase-mongodb/$extensionVersion/liquibase-mongodb-$extensionVersion.jar" -Outfile $extensionPath -UseBasicParsing
```
2. The `Function Get-DatabaseJar` function has this code:

```ps
if ([string]::IsNullOrEmpty($Version))
{
    # Get the latest version for the extension
    $downloadUrl = (Get-LatestVersionDownloadUrl -Repository $repositoryName | Where-Object {$_.EndsWith(".jar")})
}
else
{
    # Download version matching extension
    $downloadUrl = (Get-LatestVersionDownloadUrl -Repository $repositoryName -Version $liquibaseVersion | Where-Object {$_.EndsWith(".jar")})
}
```

I think the if check should be:
```ps
if ([string]::IsNullOrEmpty($liquibaseVersion))
```